main.py

```diff
@@ -226,6 +184,71 @@ def create_student_accounts(self):
             logging.error(e)
             logging.error(r.text)
 
+    @elapsed
+    @retry(**RETRY_PARAMS)
+    def load_meetings(self):
+        """
+        Load Zoom Meetings data for the prior day based on the last
+        date data is available for in the database. If no prior data
+        exists, it will pull from the start of August for the current
+        school year.
+        """
+        run_date = self.get_last_meeting_date()
+        if run_date >= datetime.datetime.today().date():
+            return
+        table_name = "Zoom_Meetings"
+        page_token = True
+        meetings = []
+        params = {
+            "page_size": 300,
+            "type": "past",
+            "from": run_date,
+            "to": run_date,
+        }
+        while page_token:
+            response = self.client.metric.list_meetings(**params).json()
+            if response.get("code") == 429:
+                logging.info("Rate limit reached; waiting 60 seconds...")
+                time.sleep(60)
+            results = response.get("meetings")
+            if results:
+                meetings.extend(results)
+            page_token = response.get("next_page_token")
+            params["next_page_token"] = page_token
+        if meetings:
+            df = pd.DataFrame(meetings)
+            self.sql.insert_into(table_name, df)
+            logging.debug(
```

Comment:
I think it would be useful to have this as `logging.info`

Reply:
I made this debug because it is very noisy/verbose. In fact, the participants one should really be debug as well. There is an env var for switching to debug mode that allows more detailed logging if needed. Data volume is ~ 1 million records per month. Which translates to very big log files.

Reply:
@susanxiong Check the latest push. I think that should address the noise but still provide the general info.

Reply:
Oh that makes sense. Ok!